tests/test_license_files.py

```diff
@@ -0,0 +1,52 @@
+"""
+Skeleton tests for `license_files` from `setup.cfg`.  This is a `setuptools`
+extension which is not yet ratified by the Python community.
+
+`license_files` is the subject of a PEP: https://peps.python.org/pep-0639/
+"""
+
+import pytest
+
+
+@pytest.mark.xfail
+def test_license_files_setupcfg_globs(project) -> None:
+    """
+    Test that file names that contain glob characters are added as globs.
+    """
+    license_files = ["LICEN[CS]E*", "COPYING*", "NOTICE*", "AUTHORS*"]
```

Comment:
I think this is the default value of the field, right? It might be better to use a non-default value so the test can tell if it's just using the default when it shouldn't.

Reply:
Yeah, good point… next time I get a moment I'll fix that.